code-for-canada/project-thundercat
# [Bug Fix] Fix spelling of name in pizza test

# Description

The name in the emails and the name in the org chart were spelled differently which caused problems with the address book.

## Type of change

- Bug fix (non-breaking change which fixes an issue)

# Testing

Manual steps to reproduce this functionality:

docker exec -it project-thundercat_backend_1 /bin/bash
python manage.py migrate custom_models zero
python manage.py migrate custom_models

1.  Try out the pizza test.
2.  Try out the address book - you should see no errors.

# Checklist

Applicable for all code changes.

~~- [ ] I have commented my code, particularly in hard-to-understand areas~~
~~- [ ] My changes generate no new accessibility errors and/or warnings~~
~~- [ ] I have added tests that prove my fix is effective or that my feature works~~
~~- [ ] I have researched WCAG2.1 accessibility standards and met them in this PR (can be navigated using a keyboard)~~
